spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/RecordFilterStrategy.java

```diff
@@ -58,4 +60,14 @@ default List<ConsumerRecord<K, V>> filterBatch(List<ConsumerRecord<K, V>> record
 		return records;
 	}
 
+	/**
+	 * Returns a boolean value that determines whether {@link FilteringBatchMessageListenerAdapter} invoke the {@link BatchMessageListener} when all {@link ConsumerRecord}
```

Comment:
Method Javadocs must be imperative, like a command what yo do.
The length of the like not more than `120`.

Reply:
Thanks for your comment.  🙇‍♂️ 
I've revised it to be more imperative, referencing other Javadocs. 
Could you take a look and see if it's okay?